include/rtl_433.h

```diff
@@ -8,12 +8,32 @@
 #include <stdint.h>
 #include "list.h"
 #include <time.h>
+#include <stdint.h>
+
+#include "rtl_433_devices.h"
+#include "bitbuffer.h"
+#include "data.h"
+
+#ifdef _WIN32
+#include <windows.h>
+#include <io.h>
+#include <fcntl.h>
+#ifdef _MSC_VER
+#include "getopt/getopt.h"
+#define F_OK 0
+#endif
+#endif
+#ifndef _MSC_VER
+#include <unistd.h>
+#include <getopt.h>
+#endif
 
 #define DEFAULT_SAMPLE_RATE     250000
 #define DEFAULT_FREQUENCY       433920000
 #define DEFAULT_HOP_TIME        (60*10)
-#define DEFAULT_ASYNC_BUF_NUMBER    0 // Force use of default value (librtlsdr default: 15)
-#define DEFAULT_BUF_LENGTH      (16 * 32 * 512) // librtlsdr default
+#define DEFAULT_HOP_EVENTS      2
+#define DEFAULT_ASYNC_BUF_NUMBER    0 // Force use of default value (was : 32)
```

Comment:
Why is this file changed? Why is the comment changed that said `// Force use of default value (librtlsdr default: 15)` to `// Force use of default value (was : 32)`? It's not obvious to me why that's changed and why the 'previous' value is now 32 instead of 15.

Reply:
Oh, does this have something to do with the merge? If so, disregard the comments I've made. It just seemed strange any changes were done to this file.